packages/arb-bridge-eth/contracts/arch/OneStepProof2.sol

```diff
@@ -477,7 +477,10 @@ contract OneStepProof2 is IOneStepProof2, OneStepProofCommon {
             handleOpcodeError(context);
             return;
         }
-        require(val2.intVal < (1 << 64), "buffer index must be 64-bit");
+        if (val2.intVal >= 1 << 64) {
```

Comment:
I would've expected this to be `1 << 64 - 7` so that the read doesn't overflow. Are we defining overflow here as 0s? Might be nicer to just disallow reading past the max size like we disallow writing past the max size

Reply:
The 8 in getBuffer8 refers to the number of bits, not the number of bytes, so that's not necessary. If you scroll down a bit to getBuffer64 you'll see we do have a `- 7` there.